roles/variables/vars/main.yml

```diff
@@ -101,8 +101,6 @@ kafka_broker:
   log4j_file: "{% if installation_method == 'archive' %}{{archive_destination_path}}/confluent-{{confluent_package_version}}{% endif %}/etc/kafka/log4j.properties"
   jaas_file: "{{ archive_config_base_path if installation_method == 'archive' else '' }}/etc/kafka/kafka_server_jaas.conf"
   rest_proxy_password_file: "{{ archive_config_base_path if installation_method == 'archive' else '' }}/etc/kafka/password.properties"
-  datadir:
-    - /var/lib/kafka/data
```

Comment:
wdyt about adding an assertion around this change? I feel customers may run into this when they are upgrading and it would be good for the playbook to see an error message telling them they need to update their inventory variables

Reply:
Thanks @domenicbove . Added a check before we proceed for any other task.